config/module_params.config

```
@@ -0,0 +1,14 @@
+// Module specific parameters are stored here defined here as parameters
```

just chiming in, this phrasing seems redudant?
```suggestion
// Module specific parameters are defined here
```